civihr_default_theme/scss/components/_panel.scss

```diff
@@ -199,12 +199,12 @@ form {
     right: 0;
     padding: 0;
     font-size: 1.1rem;
-    line-height: 34px;
+    line-height: $input-height-base;
     background: none;
     color: $crm-color-neutral;
     cursor: default;
     z-index: 4;
-    width: 34px;
+    width: $input-height-base;
```

Comment:
That does not really sound well, it either:
1) needs a comment OR
2) var needs to be renamed

Reply:
that is a default bootstrap variable, which we have used in other places also. that is the height for any form element